test/distributions/test_distributions.py

```diff
@@ -452,6 +452,10 @@ def is_all_nan(tensor):
                                         torch.randn(2, 3, 5).abs().requires_grad_()),
             'transforms': AffineTransform(1, 2),
         },
+        {
+            'base_distribution': Uniform(torch.tensor(1e8).log(), torch.tensor(1e10).log()),
+            'transforms': ExpTransform(),
+        },
```

Comment:
Not sure if this is the right place for the test. This is simply the breaking example provided in the original issue.

Reply:
This looks good!